app/models/ext_management_system.rb

```diff
@@ -628,6 +646,10 @@ def build_authentication_by_role(options)
     update_authentication(creds,options)
   end
 
+  def authtype_by_role(role)
+    role == :default ? :bearer : role
```

Comment:
This isn't the only place that describes the knowledge of the default auth type. It would be great if this could be DRY'd up